### DISCUSSION TITLE
[UI] Zoom

### DISCUSSION THREAD

**Discussion by MerryPanda** [phcode-dev/phoenix#1617] [General]
You can Zoom using CTRL+"+/-", but there is no way to return to standard zoom or see current zoom level

I have found the Zoom info, but there should be the CTRL+"0" command to easily switch to the standard, imho

**Comment by JuriTuuti** [comment]
Also, I would much prefer if Ctrl-+/- increased/decreased the editor font size as it did in Brackets, and shift-Crtl-+/- the UI font size.

Or make these shortcuts overrideable.  For some reason (3.7.3) does not allow override them.

**Reply by abose** [reply]
Have you tried https://docs.phcode.dev/docs/Features/keyboard-shortcuts/ to change shortcuts @JuriTuuti

**Reply by JuriTuuti** [reply]
I tired that, but _Ctrl-**+**/**-**_ and _Ctrl-Shift-**+**/**-**_ are not editable.   Trying to null them does write to keymap override, but generates error.

This is probably Phoenix issue.  And frankly, when I was testing and spent time trying to fix it, it seemed like an issue.  But... It is not often I am zooming in/out or change the font size on the editor after it has been properly set up, so it really is a Very Minor Inconvenience, and nothing to worry about. Getting the UI zoom feature is well worth it.

**Reply by abose** [reply]
I have just fixed some of the errors setting shirtcuts. But as i checked, it looks like ctrl +/- is not reassignable in phcode as it is a browser defined shortcut that cant be changed.
